docs/Security.rst

```diff
@@ -658,6 +661,45 @@ configuration :
             ...
 
 
+.. _Security.SameSiteConfiguration:
+
+======================
+SameSite Configuration
+======================
+
+SameSite is a property set on HTTP cookies. It can prevent some CSRF attacks.
+SameSite property can take one of theses values : None, Strict, and Lax
+
+With value **None**, when a request is done on eXo Server, there is no verification on the referer. The cookies is used.
+For example, if a user receives a malicious email, containing a link forged to call a REST endpoint altering data, like deleting a space, changing a permission, etc..
```

Comment:
```suggestion
For example, when a user receives a malicious email, containing a link forged to call a dans
data-altering REST endpoint such as deleting a space, changing a permission, etc..
```

Reply:
Is there a typo in your suggestion ? the word "dans" should be removed, right ?

```suggestion
For example, when a user receives a malicious email, containing a link forged to call a data-altering REST endpoint altering data, such as deleting a space, changing a permission, etc..
```

Reply:
yes but tour suggestion still has a repetition for data altering. I updated mine. please review